### PR TITLE
VPC: update default network offering for vpc tier to conserve_mode=1

### LINKED ISSUE #8313: VPC: Public IP cannot be used by multiple services
Steps to reproduce the issue
- create a vpn, a tier and a vm
- acquire a public IP
- create port forwarding rule to the vm
- create load balancer rule via cmk/cloudmonkey

Expected:
all good

Actual result:
```
(localcloud) 🐱 > create loadbalancerrule publicipid=712e9718-7ca7-4866-a930-516919098201 publicport=80 privateport=80 algorithm=roundrobin name=http
🙈 Error: (HTTP 530, error code 4250) Failed to create load balancer rule: http
```
the error logs show
```
2023-12-06 11:04:11,098 WARN  [c.c.n.l.LoadBalancingRulesManagerImpl] (qtp249034932-13:ctx-a267cf2c ctx-045bb55f ctx-fcc51a3c) (logid:e9fde666) Failed to create load balancer due to
java.security.InvalidParameterException: The IP 10.0.53.125 is already used as PortForwarding rather than Lb
        at com.cloud.network.NetworkModelImpl.canIpUsedForNonConserveService(NetworkModelImpl.java:422)
        at com.cloud.network.NetworkModelImpl.checkIpForService(NetworkModelImpl.java:1621)
```

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~

~~~

##### CLOUDSTACK VERSION

~~~
4.18/4.19
~~~

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR updates the conserve mode of default vpc tier offering to conserve_mode=1
so we can create both port forwarding and load balancing rules on a public IP in vpc tiers.

This fixes #8313 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan package

File: server/src/main/java/com/cloud/network/vpc/VpcManagerImpl.java [hunk 1]
```diff
@@ -1809,9 +1809,9 @@ public void validateNtwkOffForVpc(final NetworkOffering guestNtwkOff, final List
          * ("No redunant router support when network belnogs to VPC"); }
          */
 
-        // 4) Conserve mode should be off
+        // 4) Conserve mode should be off in elder versions
```
Comment (Pearl1594) [inline server/src/main/java/com/cloud/network/vpc/VpcManagerImpl.java:1812]:
```suggestion
        // 4) Conserve mode should be off in older versions
```

**Comment by weizhouapache** [comment]
@blueorangutan test

**Comment by harikrishna-patnala** [comment]
@blueorangutan test

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan test matrix

**Comment by alexandremattioli** [comment]
@weizhouapache great idea, conserve mode should definitely be true by default

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan test matrix

**Comment by weizhouapache** [comment]
this is also a simple PR, users can benefit from it
cc @shwstppr @DaanHoogland

**Comment by shwstppr** [comment]
@blueorangutan package

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Review by Pearl1594 — APPROVED** [review]

**Comment by shwstppr** [comment]
Results from the backend,

```
14:34:24 + MSG='{"body": "<b>[SF] Trillian test result (tid-8613)</b>\nEnvironment: kvm-centos7 (x2), Advanced Networking with Mgmt server 7\nTotal time taken: 52104 seconds\nMarvin logs: [https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr8309-t8613-kvm-centos7.zip\nSmoke](https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr8309-t8613-kvm-centos7.zip/nSmoke) tests completed. 121 look OK, 0 have errors, 0 did not run\nOnly failed and skipped tests results shown below:\n\n\nTest | Result | Time (s) | Test File\n--- | --- | --- | ---\n"}'
```

**Pull request merged**